replicated_kv_store/states.go

```diff
@@ -315,3 +320,69 @@ func (node *RaftNode) StartElection() {
 
 	go node.RunElectionTimer() // begin the timer during which this candidate waits for votes
 }
+
+func (node *RaftNode) Writeto() {
+	for range node.newCommitReadyChan {
+		// Find which entries we have to apply.
+		node.raft_node_mutex.Lock()
+		//savedTerm := node.currentTerm
+		//savedLastApplied := node.lastApplied
+		var entries []protos.LogEntry
+		if node.commitIndex > node.lastApplied {
+			entries = node.log[node.lastApplied+1 : node.commitIndex+1]
+			node.lastApplied = node.commitIndex
+		}
+		node.raft_node_mutex.Unlock()
+
+		for _, entry := range entries {
+			formData := url.Values{
+				"value": {entry.Operation[2]},
+			}
+
+			timeout := time.Duration(5 * time.Second)
```

Comment:
5 seconds is too much since we are interacting within the local machine.. you can keep it like 100ms